resources/linux/rpm/code.spec.template

```diff
@@ -9,6 +9,7 @@ License:  MIT
 URL:      https://code.visualstudio.com/
 Icon:     @@NAME@@.xpm
 Requires: glibc >= 2.15
+Requires: GConf2
```

Comment:
I think we actually want to do what we do for `libXss.so.1` here and have a separate one for 64-bit and 32-bit:

```
libgconf-2.so.4

libgconf-2.so.4()(64bit)
```

Chromium seems to do it this way in https://cs.chromium.org/chromium/src/chrome/installer/linux/rpm/expected_deps_x86_64?dr=C&sq=package:chromium